### DISCUSSION TITLE
Question about battery percentage

### DISCUSSION THREAD

**Discussion by jrhbcn** [PricelessToolkit/MailBoxGuard#23] [Q&A]
Hi,

I am waiting for some replacement hardware components to rebuild my sensor (see #20 ;). Meantime, I've started to look at the code and I see in the arduino sensor code that the battery percentage is mapped from 3.6 to 4.2 volts:

`float perc = map(volts * 1000, 3600, 4200, 0, 100);`

However, the low battery code is sent when voltage is below 3.36:

` if (volts < 3.36 and loopcounter == 1 ){   // Don't change "3.36" !!`

Is there any reason for this? Shouldn't the percentage follows around the same 3.36-4.2 voltages?

**Comment by PricelessToolkit** [comment]
You're right!, I changed it.

**Comment by jrhbcn** [comment]
Perfect!
